pom.xml

```diff
@@ -160,6 +166,34 @@
       <artifactId>objenesis</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <!-- Jenkins 2.190.1 requires this newer version -->
+      <groupId>org.slf4j</groupId>
+      <artifactId>log4j-over-slf4j</artifactId>
+      <version>1.7.26</version>
```

Comment:
FYI there's actually a property `slf4j.version` to set this for all slf4j*

Reply:
I couldn't find the property `slf4j.version` in bom 2.138.x.  That didn't surprise me too much, since it was not needed (at least in this case) until 2.190.1.

Reply:
It is in `plugin-pom` not `bom`. And it should not be necessary to override but we are waiting on https://github.com/jenkinsci/plugin-pom/pull/229.